doc/manual/src/installation/installing-binary.md

```diff
@@ -175,6 +176,16 @@ and `/etc/zshrc` which you may remove.
    This will remove the Nix Store volume and everything that was added to the
    store.
 
+   If this command indicates that it couldn't remove the volume, your Nix Store
+   volume may not be mounted. Run the following to double-check:
+
+   ```console
+   diskutil list
+   ```
+
+   If you see a Nix Store volume, re-run the diskutil deleteVolume command, but
```

Comment:
This explains what to do if the volume is indeed mounted but not what to do if it's not. Probably as simple as mounting it but maybe there's some help we can offer here on how to figure out how to find the right volume to mount?

Reply:
Ah, I was the one who misunderstood. The "to double-check" made it sound to me like "it's probably mounted but the label may differ, use diskutil list to figure out the volume and used the bsd identifier."

Maybe it can say something along the lines of, "If there's no Nix Store volume, you can skip this step. Otherwise use the identifier to delete it?"

Reply:
I've force-pushed some phrasing tweaks. WDYT?

Reply:
Perfect!